bin/mark_outliers_in_json

```diff
@@ -21,11 +21,13 @@
 """
 
 import argparse
-import bz2
-import json
-import numpy
 import os
 import os.path
+import sys
+
+sys.path.insert(0, '../warmup')
```

Comment:
Didn't Python have some relative import syntax which we could use (I never used it)?